torch/_inductor/triton_bundler.py

```diff
@@ -231,8 +233,9 @@ def read_and_emit(
                             )
                         file.write(payload)
                     counters["inductor"]["triton_bundler_read_and_emit_kernel"] += 1
-                    if artifact.filename.endswith(".cubin"):
-                        # Each kernel has bunch of files like .cubin, .json, .ttir
+                    file_extension = os.path.splitext(artifact.filename)
```

Comment:
```suggestion
                    _, file_extension = os.path.splitext(artifact.filename)
```

Reply:
`os.path.splitext` will return a tuple including `root` and `ext`, refer to https://docs.python.org/3/library/os.path.html#os.path.splitext